pkg/test/framework/test.go

```diff
@@ -204,8 +208,7 @@ func (t *Test) runInternal(fn func(ctx TestContext), parallel bool) {
 	}
 	suiteName := t.s.settings.TestID
 	if len(t.featureLabels) < 1 && !features.GlobalWhitelist.Contains(suiteName, myGoTest.Name()) {
-
-		myGoTest.Fatalf("Detected new test %s in suite %s with no feature labels.  "+
+		myGoTest.Logf("Detected new test %s in suite %s with no feature labels.  "+
```

Comment:
What is the reason for this change?  From docs, it looks like this will not fail the related tests, which is the desired outcome..

Reply:
This wasn't actually intended to be merged, but I got frustrated because my options are to either disable this, merge https://github.com/istio/istio/pull/24875, add some bogus feature, or spend an hour adding whitelists.... its a horrible dev UX without the linked PR

Reply:
I have reverted this